gerardo-santos/react-apprenticeship-capstone-2
# feat: added styles, title, and date-picker

## Description
```
The date picker was added

- Styles, title, and date picker were added.

```
## Type of Change

```
Delete options that are not relevant.
```



## How should this be tested?
```
Add a brief description of how can be tested this deliverable step by step. 

- Make sure the title and the date picker appear on the screen.

```
## Any background context you want to provide?
```
N/A
```
## What is the deliverable?
```
PR with title and date picker.
```
## Screenshots and links

![image](https://user-images.githubusercontent.com/97237961/150428743-06d9efa0-ef60-4580-86eb-9fcb8e930ee0.png)


## Notes
```
Please provide any additional information or any concern you may want to be reviewed more in detail
```
## Checklist
N/A
